src/Sylius/Behat/Context/Setup/AdminUserContext.php

```diff
@@ -73,4 +74,17 @@ public function thereIsAnAdministratorWithName($username)
         $this->userRepository->add($adminUser);
         $this->sharedStorage->set('administrator', $adminUser);
     }
+
+    /**
+     * @Given /^(this administrator) is using ("[^"]+" locale)$/
+     * @Given /^(I) am using ("[^"]+" locale) for my panel$/
+     */
+    public function thisAdministratorIsUsingLocale(AdminUserInterface $adminUser, $localeCode)
+    {
+        $adminUser->setLocaleCode($localeCode);
+
+        $this->userRepository->add($adminUser);
+        $this->sharedStorage->set('administrator', $adminUser);
+        $this->sharedStorage->set('admin', $adminUser);
```

Comment:
Inb4 any questions what is going on and why, we use both of those tags in setup/transformer contexts, and since i'm both of them at the moment i can't be sure which one might pop up.
Hence this little workaround, until a proper fix will be merged.